src/test/java/com/es/phoneshop/model/cart/DefaultCartServiceTest.java

```diff
@@ -54,4 +54,10 @@ public void AddToCartCorrectQuantity() {
         cartService.add(cart, 0l, 1);
         assertThat(cart, notNullValue());
     }
+
+    @Test
+    public void DeleteItem() {
```

Comment:
Please add tests, update method is not even covered, recalculate functionality is not tested